### PR TITLE
71288: Add mapping functionality for mapping GOB events to the legacy model

### PR DESCRIPTION
**Pull request by JKoetsier**
Apologies for all the reformatting in the test and alembic files. I ran Black and isort on the src directory. That took these files too. Luckily they're all together, so easy to skip through :)

### CODE REVIEW

File: src/gobeventproducer/mapper.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        def get_value(eventdata: dict, fieldmapping: FieldMappingTypes):
+            if isinstance(fieldmapping, MappedObjectDefinition):
+                return {
+                    newkey: get_value(eventdata, oldkey_or_definition)
+                    for newkey, oldkey_or_definition in fieldmapping.mapping.items()
+                }
+            elif isinstance(fieldmapping, str):
+                return eventdata.get(fieldmapping)
+
```
Comment (roel-devries) [inline src/gobeventproducer/mapper.py:43]: else raise Exception?

File: src/gobeventproducer/mapping/__init__.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+class MappingDefinitionLoader:
+    """Entry class for fetching a MappingDefinition."""
+
+    _mapping_definitions = None
+
+    def __init__(self):
+        if not self._mapping_definitions:
+            self._load()
+
+    def _load_mapping_definition(self, path: Path):
```
Comment (roel-devries) [inline src/gobeventproducer/mapping/__init__.py:37]: return type

File: src/gobeventproducer/mapper.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        return eventdata
+
+
+class EventDataMapper(BaseEventDataMapper):
+    """Map the internal GOB data to an event for a specific Amsterdam Schema version."""
+
+    def __init__(self, mapping_definition: MappingDefinition):
+        self.mapping_definition = mapping_definition
+
+    def map(self, eventdata: dict):
```
Comment (roel-devries) [inline src/gobeventproducer/mapper.py:32]: return type

File: src/gobeventproducer/mapper.py (hunk 1 above)
Comment (JKoetsier) [reply src/gobeventproducer/mapper.py:43]: Ah yes, should have added that one, will do :)

**Review by roel-devries — APPROVED** [review]

**Pull request merged**